flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionGraphRescalingTest.java

```diff
@@ -58,7 +58,8 @@ public void testExecutionGraphArbitraryDopConstructionTest() throws Exception {
                 createVerticesForSimpleBipartiteJobGraph(initialParallelism, maxParallelism);
         final JobGraph jobGraph = new JobGraph(jobVertices);
 
-        ExecutionGraph eg = TestingExecutionGraphBuilder.newBuilder().setJobGraph(jobGraph).build();
```

Comment:
should this test (and others) be renamed to refer to the `DefaultExecutionGraph`?